python/feldera-api-client/feldera_api_client/models/update_connector_request.py

```diff
@@ -13,36 +13,37 @@
 
 @define
 class UpdateConnectorRequest:
-    """Request to update an existing data-connector.
+    """Request to update an existing connector.
 
     Attributes:
-        description (str): New connector description.
-        name (str): New connector name.
         config (Union[Unset, None, ConnectorConfig]): A data connector's configuration
+        description (Union[Unset, None, str]): New connector description. If absent, existing name will be kept
```

Comment:
is it worth repeating the type in the comments?
It is visible just below. 
Makes it harder to maintain.

Reply:
This code is generated.